### DISCUSSION TITLE
How to identify security vulnerability is getting fixed without updating chart

### DISCUSSION THREAD

**Discussion by Kartikey-star** [artifacthub/hub#2066] [Q&A] [answered]
Hi,
I have been notified that my chart has a security vulnerability due to a go package. I have bumped the version to the prescribed version ,but how can i be sure that the cve is resolved without updating my chart in artifact hub repo.

**Comment by tegioz** [comment, marked as answer]
Hi @Kartikey-star 👋

Artifact Hub uses Trivy to generate the security reports. You can download it and run it locally to make sure everything is all right before updating your chart 🙂

**Reply by Kartikey-star** [reply]
Thanks @tegioz. That was really helpful .
